testdata/baselines/reference/submodule/compiler/checkJsTypeDefNoUnusedLocalMarked.symbols.diff

```diff
@@ -7,5 +7,5 @@
 ->module.exports : Symbol(module.exports, Decl(something.js, 0, 0))
 ->module : Symbol(export=, Decl(something.js, 0, 0))
 +>module.exports : Symbol(export=, Decl(something.js, 0, 0))
-+>module : Symbol(module.exports)
++>module : Symbol("something", Decl(something.js, 0, 0))
```

Comment:
Kind of funky that before, `module.exports` was `module.exports` and `module` was `export=`, but now `module.exports` is `export=` and `module` is module name...

Reply:
I think the new behavior is defensible. The old one--not so much.